js_modules/dagster-ui/packages/ui-core/src/runs/ReexecutionDialog.tsx

```diff
@@ -221,6 +222,13 @@ export const ReexecutionDialog = (props: Props) => {
         return (
           <Group direction="column" spacing={16}>
             <div>{message()}</div>
+            {isPartOfBackfill ? (
+              <div>
+                One or more of these runs is part of a backfill. If the backfill has completed,
+                re-executing these runs will not update the backfill status or launch runs of
+                downstream dependencies.
+              </div>
+            ) : undefined}
```

Comment:
Would it be useful to link to the relevant backfill from here?

Reply:
Ahh that would be cool. I think there's a case here where your selection can include runs from several backfills, but I'll make it link to the backfill in the common case where there's just a single backfill ID. Seems like it'd make checking the status much easier